datasetviewer/__init__.py

```diff
@@ -1 +1,17 @@
+
 name = "datasetviewer"
+
+def plot(ds):
```

Comment:
Not sure about this OrderdDict check. For example see this, which should work....
```
In [1]: import xarray as xr                                                                                      

In [2]: import numpy as np                                                                                       

In [3]: ds = xr.Dataset({'foo': np.arange(10), 'bar': ('x', [1, 2]), 'baz': np.pi})                              

In [4]: type(ds)                                                                                                 
Out[4]: xarray.core.dataset.Dataset

In [5]: import datasetviewer                                                                                     

In [6]: datasetviewer.plot(ds)                                                                                   
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-6-56411fac217d> in <module>
----> 1 datasetviewer.plot(ds)

~/sources/dataset_viewer/datasetviewer/__init__.py in plot(ds)
     10 
     11     if type(ds) is not collections.OrderedDict:
---> 12         raise ValueError("Error: Argument is not of type Dataset.")
     13 
     14     QAPP = QtWidgets.QApplication(sys.argv)

ValueError: Error: Argument is not of type Dataset.
```